include/GafferOSL/OSLCode.h

```diff
@@ -72,6 +72,8 @@ class OSLCode : public OSLShader
 		/// combo we use everywhere else.
 		ShaderCompiledSignal &shaderCompiledSignal();
 
+		virtual void loadShader( const std::string &shaderName, bool keepExistingValues=false );
```

Comment:
I assume if I'm adding the custom serializer for OSLCode as you mentioned below, I can just remove this again?